### PR TITLE
Add landing message. #1

### PR DESCRIPTION
**Pull request by bachns**
I have just completed it.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,6 +6,6 @@
     <title>Document</title>
 </head>
 <body>
-    
+    <h1>Welcome to Potionfy! Enjoy your stay.</h1>
```
Comment (nullbyte0x0000) [inline index.html:9]: Could you remove the `Enjoy your stay.` string and make the `Welcome to Potionfy!` string red?

Comment (bachns) [reply index.html:9]: So, can do!

Comment (bachns) [reply index.html:9]: Ok !

**Review by nullbyte0x0000 — CHANGES_REQUESTED** [review]
Please, change the code as per your comment on the review

**Review by nullbyte0x0000 — APPROVED** [review]

**Pull request merged**